### PR TITLE
Various code clean up, fix test flake

### PR DESCRIPTION
**Pull request by martinpitt**

### CODE REVIEW

File: src/libvirtApi/common.js [hunk 1]
```diff
@@ -161,88 +161,91 @@ function doUsagePolling(name, connectionName, objPath) {
...
-            .catch(ex => {
-                console.warn("NodeGetMemoryStats failed:", ex.toString());
-                return Promise.reject(ex);
-            });
+    try {
+        const [stats] = await call(connectionName, "/org/libvirt/QEMU", "org.libvirt.Connect", "NodeGetMemoryStats", [-1, 0], { timeout, type: "iu" });
+        store.dispatch(setNodeMaxMemory({ memory: stats.total }));
+    } catch (ex) {
+        console.warn("NodeGetMemoryStats failed:", ex.toString());
+        throw ex;
```
Comment (cockpituous) [inline src/libvirtApi/common.js:217]: These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-053137-bfbdf98b-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/common.js [hunk 2]
```diff
@@ -161,88 +161,91 @@ function doUsagePolling(name, connectionName, objPath) {
...
-            .catch(ex => {
-                console.error(`get os list returned error: "${JSON.stringify(ex)}"`);
-                parseOsInfoList('[]');
-            });
+    try {
+        const osList = await python.spawn(getOSListScript, null, { err: "message", environ: ['LC_ALL=C.UTF-8'] });
+        parseOsInfoList(osList);
+    } catch (ex) {
+        console.error(`get os list returned error: "${JSON.stringify(ex)}"`);
+        parseOsInfoList('[]');
```
Comment (cockpituous) [inline src/libvirtApi/common.js:228]: These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-053137-bfbdf98b-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/common.js [hunk 3]
```diff
@@ -161,88 +161,91 @@ function doUsagePolling(name, connectionName, objPath) {
...
-                else // Transient network which got undefined when stopped
-                    return store.dispatch(undefineNetwork({ connectionName, id: netPath }));
-            })
-            .catch(ex => console.warn("networkUpdateOrDelete action failed:", ex.toString()));
+async function networkUpdateOrDelete(connectionName, netPath) {
+    try {
+        const [objPaths] = await call(connectionName, "/org/libvirt/QEMU", "org.libvirt.Connect", "ListNetworks", [0],
+                                      { timeout, type: "u" });
+        if (objPaths.includes(netPath))
+            return networkGet({ connectionName, id: netPath, updateOnly: true });
```
Comment (cockpituous) [inline src/libvirtApi/common.js:237]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-053137-bfbdf98b-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/common.js [hunk 4]
```diff
@@ -161,88 +161,91 @@ function doUsagePolling(name, connectionName, objPath) {
...
+        else // Transient network which got undefined when stopped
+            store.dispatch(undefineNetwork({ connectionName, id: netPath }));
+    } catch (ex) {
+        console.warn("networkUpdateOrDelete action failed:", ex.toString());
```
Comment (cockpituous) [inline src/libvirtApi/common.js:241]: These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-053137-bfbdf98b-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/common.js [hunk 5]
```diff
@@ -418,15 +421,16 @@ function startEventMonitorStoragePools(connectionName) {
...
-            .catch(ex => console.warn("storagePoolStopOrUndefine action failed:", ex.toString()));
+async function storagePoolStopOrUndefine(connectionName, poolPath) {
+    try {
+        const [objPaths] = await call(connectionName, "/org/libvirt/QEMU", "org.libvirt.Connect", "ListStoragePools", [0], { timeout, type: "u" });
+        if (objPaths.includes(poolPath))
+            await storagePoolGet({ connectionName, id: poolPath, updateOnly: true });
+        else // Transient pool which got undefined when stopped
+            store.dispatch(undefineStoragePool({ connectionName, id: poolPath }));
+    } catch (ex) {
+        console.warn("storagePoolStopOrUndefine action failed:", ex.toString());
```
Comment (cockpituous) [inline src/libvirtApi/common.js:432]: These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-053137-bfbdf98b-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/domain.js [hunk 6]
```diff
@@ -685,20 +680,20 @@ export async function domainGet({
...
-            });
+export async function domainGetAll({ connectionName }) {
+    try {
+        const [objPaths] = await call(connectionName, '/org/libvirt/QEMU', 'org.libvirt.Connect', 'ListDomains', [0],
+                                      { timeout, type: 'u' });
+        store.dispatch(deleteUnlistedVMs(connectionName, [], objPaths));
+        await Promise.all(objPaths.map(path => domainGet({ connectionName, id: path })));
+    } catch (ex) {
+        console.warn('GET_ALL_VMS action failed:', ex.toString());
+        throw ex;
```
Comment (cockpituous) [inline src/libvirtApi/domain.js:695]: These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-053137-bfbdf98b-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/interface.js [hunk 7]
```diff
@@ -33,42 +33,43 @@ import { call, Enum, timeout } from './helpers.js';
...
-            .then(xml => {
-                const iface = parseIfaceDumpxml(xml);
-                return store.dispatch(updateOrAddInterface(Object.assign({}, props, iface)));
-            })
-            .catch(ex => console.log('listInactiveInterfaces action for path', objPath, ex.toString()));
+        const [xml] = await call(connectionName, objPath, 'org.libvirt.Interface', 'GetXMLDesc', [0], { timeout, type: 'u' });
+        const iface = parseIfaceDumpxml(xml);
+        store.dispatch(updateOrAddInterface(Object.assign(props, iface)));
+    } catch (ex) {
+        console.log('listInactiveInterfaces action for path', objPath, ex.toString());
```
Comment (cockpituous) [inline src/libvirtApi/interface.js:61]: These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-053137-bfbdf98b-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/interface.js [hunk 8]
```diff
@@ -33,42 +33,43 @@ import { call, Enum, timeout } from './helpers.js';
...
-            .catch(ex => {
-                console.warn('getAllInterfaces action failed:', ex.toString());
-                return Promise.reject(ex);
-            });
+    try {
+        const [ifaces] = await call(connectionName, '/org/libvirt/QEMU', 'org.libvirt.Connect', 'ListInterfaces', [flags], { timeout, type: 'u' });
+        await Promise.all(ifaces.map(path => interfaceGet({ connectionName, id: path })));
+    } catch (ex) {
+        console.warn('getAllInterfaces action failed:', ex.toString());
+        throw ex;
```
Comment (cockpituous) [inline src/libvirtApi/interface.js:73]: These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-053137-bfbdf98b-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/network.js [hunk 9]
```diff
@@ -62,64 +62,61 @@ export function networkDeactivate({ connectionName, objPath }) {
...
-            .catch(ex => {
-                console.warn('GET_ALL_NETWORKS action failed:', ex.toString());
-                return Promise.reject(ex);
-            });
+    try {
+        const [objPaths] = await call(connectionName, '/org/libvirt/QEMU', 'org.libvirt.Connect', 'ListNetworks', [0], { timeout, type: 'u' });
+        await Promise.all(objPaths.map((path) => networkGet({ connectionName, id: path })));
+    } catch (ex) {
+        console.warn('GET_ALL_NETWORKS action failed:', ex.toString());
+        throw ex;
```
Comment (cockpituous) [inline src/libvirtApi/network.js:73]: These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-053137-bfbdf98b-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/network.js [hunk 10]
```diff
@@ -62,64 +62,61 @@ export function networkDeactivate({ connectionName, objPath }) {
...
+        if ("Name" in resultProps)
+            props.name = resultProps.Name.v.v;
+        props.id = objPath;
+        props.connectionName = connectionName;
+
+        const [xml] = await call(connectionName, objPath, 'org.libvirt.Network', 'GetXMLDesc', [0], { timeout, type: 'u' });
+        const network = parseNetDumpxml(xml);
+        store.dispatch(updateOrAddNetwork(Object.assign(props, network), updateOnly));
+    } catch (ex) {
+        console.warn('GET_NETWORK action failed for path', objPath, ex.toString());
```
Comment (cockpituous) [inline src/libvirtApi/network.js:109]: These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-053137-bfbdf98b-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/nodeDevice.js [hunk 11]
```diff
@@ -33,47 +33,44 @@ import { call, timeout } from './helpers.js';
...
-                                return store.dispatch(updateOrAddNodeDevice(deviceXmlObject));
-                            });
-                } else {
-                    return store.dispatch(updateOrAddNodeDevice(deviceXmlObject));
-                }
-            })
-            .catch(ex => console.warn('GET_NODE_DEVICE action failed for path', objPath, ex.toString()));
+        store.dispatch(updateOrAddNodeDevice(deviceXmlObject));
+    } catch (ex) {
+        console.warn('GET_NODE_DEVICE action failed for path', objPath, ex.toString());
```
Comment (cockpituous) [inline src/libvirtApi/nodeDevice.js:60]: These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-053137-bfbdf98b-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/nodeDevice.js [hunk 12]
```diff
@@ -33,47 +33,44 @@ import { call, timeout } from './helpers.js';
...
-            .catch(ex => {
-                console.warn('GET_ALL_NODE_DEVICES action failed:', ex.toString());
-                return Promise.reject(ex);
-            });
+    try {
+        const [objPaths] = await call(connectionName, '/org/libvirt/QEMU', 'org.libvirt.Connect', 'ListNodeDevices', [0], { timeout, type: 'u' });
+        await Promise.all(objPaths.map(path => nodeDeviceGet({ connectionName, id: path })));
+    } catch (ex) {
+        console.warn('GET_ALL_NODE_DEVICES action failed:', ex.toString());
+        throw ex;
```
Comment (cockpituous) [inline src/libvirtApi/nodeDevice.js:72]: These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-053137-bfbdf98b-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/snapshot.js [hunk 13]
```diff
@@ -39,78 +39,55 @@ export function snapshotCurrent({ connectionName, objPath }) {
...
-                }));
-            });
+        const snaps = [];
+        snapXmlList.forEach(snap => {
+            if (snap.status === 'fulfilled') {
+                const snapParams = parseDomainSnapshotDumpxml(snap.value.xml);
+                snapParams.isCurrent = snap.value.isCurrent;
+                snaps.push(snapParams);
+            } else {
+                console.warn("DomainSnapshot method GetXMLDesc failed", snap.reason.toString());
```
Comment (cockpituous) [inline src/libvirtApi/snapshot.js:66]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-053137-bfbdf98b-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/snapshot.js [hunk 14]
```diff
@@ -39,78 +39,55 @@ export function snapshotCurrent({ connectionName, objPath }) {
...
+            }
+        });
+        store.dispatch(updateDomainSnapshots({
+            connectionName,
+            domainPath,
+            snaps: snaps.sort((a, b) => a.creationTime - b.creationTime)
+        }));
+    } catch (ex) {
+        if (ex.name === 'org.freedesktop.DBus.Error.UnknownMethod')
+            logDebug("LIST_DOMAIN_SNAPSHOTS action failed for domain", domainPath, ", not supported by libvirt-dbus");
```
Comment (cockpituous) [inline src/libvirtApi/snapshot.js:76]: These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-053137-bfbdf98b-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/snapshot.js [hunk 15]
```diff
@@ -39,78 +39,55 @@ export function snapshotCurrent({ connectionName, objPath }) {
...
+        else
+            console.warn("LIST_DOMAIN_SNAPSHOTS action failed for domain", domainPath, ":", JSON.stringify(ex), "name:", ex.name);
+        store.dispatch(updateDomainSnapshots({
+            connectionName,
+            domainPath,
+            snaps: -1
```
Comment (cockpituous) [inline src/libvirtApi/snapshot.js:82]: These 5 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-053137-bfbdf98b-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/storagePool.js [hunk 16]
```diff
@@ -43,19 +43,19 @@ export function storagePoolCreate({
...
+    let storagePoolPath;
+    try {
+        [storagePoolPath] = await call(connectionName, '/org/libvirt/QEMU', 'org.libvirt.Connect', 'StoragePoolDefineXML',
+                                       [poolXmlDesc, 0], { timeout, type: 'su' });
+        const args = ['org.libvirt.StoragePool', 'Autostart', cockpit.variant('b', autostart)];
+
+        await call(connectionName, storagePoolPath, 'org.freedesktop.DBus.Properties', 'Set', args, { timeout, type: 'ssv' });
+    } catch (ex) {
+        if (storagePoolPath)
+            storagePoolUndefine({ connectionName, objPath: storagePoolPath });
```
Comment (cockpituous) [inline src/libvirtApi/storagePool.js:56]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-053137-bfbdf98b-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/storagePool.js [hunk 17]
```diff
@@ -68,79 +68,75 @@ export function storagePoolDeactivate({ connectionName, objPath }) {
...
-                return Promise.reject(ex);
-            });
+    try {
+        const [objPaths] = await call(connectionName, '/org/libvirt/QEMU', 'org.libvirt.Connect', 'ListStoragePools', [0],
+                                      { timeout, type: 'u' });
+        return await Promise.all(objPaths.map(async path => {
+            const [active] = await call(connectionName, path, 'org.freedesktop.DBus.Properties', 'Get',
+                                        ['org.libvirt.StoragePool', 'Active'], { timeout, type: 'ss' });
+            if (active.v)
+                return storagePoolRefresh({ connectionName, objPath: path });
```
Comment (cockpituous) [inline src/libvirtApi/storagePool.js:115]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-053137-bfbdf98b-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/storagePool.js [hunk 18]
```diff
@@ -68,79 +68,75 @@ export function storagePoolDeactivate({ connectionName, objPath }) {
...
+            else
+                return storagePoolGet({ connectionName, id: path });
```
Comment (cockpituous) [inline src/libvirtApi/storagePool.js:117]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-053137-bfbdf98b-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/storagePool.js [hunk 19]
```diff
@@ -68,79 +68,75 @@ export function storagePoolDeactivate({ connectionName, objPath }) {
...
+        }));
+    } catch (ex) {
+        console.warn('GET_ALL_STORAGE_POOLS action failed:', ex.toString());
+        throw ex;
```
Comment (cockpituous) [inline src/libvirtApi/storagePool.js:121]: These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-053137-bfbdf98b-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/storagePool.js [hunk 20]
```diff
@@ -68,79 +68,75 @@ export function storagePoolDeactivate({ connectionName, objPath }) {
...
-    return cockpit.spawn(
-        ["virsh", "-c", "qemu:///" + connectionName, "pool-capabilities"],
-        opts
-    ).then(poolCapabilities => parsePoolCapabilities(poolCapabilities), ex => {
+    try {
+        const poolCapabilities = await cockpit.spawn(
+            ["virsh", "-c", "qemu:///" + connectionName, "pool-capabilities"],
+            opts);
+        return parsePoolCapabilities(poolCapabilities);
+    } catch (ex) {
```
Comment (cockpituous) [inline src/libvirtApi/storagePool.js:136]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-053137-bfbdf98b-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/storagePool.js [hunk 21]
```diff
@@ -68,79 +68,75 @@ export function storagePoolDeactivate({ connectionName, objPath }) {
...
         console.warn('virsh pool-capabilities failed:', ex.toString());
-        return Promise.reject(ex);
-    });
+        throw ex;
```
Comment (cockpituous) [inline src/libvirtApi/storagePool.js:138]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-053137-bfbdf98b-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/storageVolume.js [hunk 22]
```diff
@@ -55,62 +52,36 @@ export function storageVolumeCreateAndAttach({
...
-                    return Promise.resolve(volumes);
-                });
-            })
-            .catch(ex => console.warn("GET_STORAGE_VOLUMES action failed for pool", poolName, ":", ex.toString()));
+        const volumeXmlList = await Promise.allSettled(storageVolumesPropsPromises);
+        return volumeXmlList
+                .filter(vol => vol.status === 'fulfilled')
+                .map(vol => parseStorageVolumeDumpxml(connectionName, vol.value[0]));
+    } catch (ex) {
+        console.warn("GET_STORAGE_VOLUMES action failed for pool", poolName, ":", ex.toString());
```
Comment (cockpituous) [inline src/libvirtApi/storageVolume.js:85]: These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-053137-bfbdf98b-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/common.js (hunk 1 above)
Comment (cockpituous) [inline src/libvirtApi/common.js:217]: These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-075343-fbf2cd7c-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/common.js (hunk 2 above)
Comment (cockpituous) [inline src/libvirtApi/common.js:228]: These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-075343-fbf2cd7c-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/common.js (hunk 4 above)
Comment (cockpituous) [inline src/libvirtApi/common.js:241]: These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-075343-fbf2cd7c-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/common.js (hunk 5 above)
Comment (cockpituous) [inline src/libvirtApi/common.js:432]: These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-075343-fbf2cd7c-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/domain.js [hunk 23]
```diff
@@ -685,20 +683,20 @@ export async function domainGet({
...
```
Comment (cockpituous) [inline src/libvirtApi/domain.js:698]: These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-075343-fbf2cd7c-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/interface.js (hunk 7 above)
Comment (cockpituous) [inline src/libvirtApi/interface.js:61]: These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-075343-fbf2cd7c-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/interface.js (hunk 8 above)
Comment (cockpituous) [inline src/libvirtApi/interface.js:73]: These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-075343-fbf2cd7c-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/network.js (hunk 9 above)
Comment (cockpituous) [inline src/libvirtApi/network.js:73]: These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-075343-fbf2cd7c-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/nodeDevice.js (hunk 11 above)
Comment (cockpituous) [inline src/libvirtApi/nodeDevice.js:60]: These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-075343-fbf2cd7c-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/nodeDevice.js (hunk 12 above)
Comment (cockpituous) [inline src/libvirtApi/nodeDevice.js:72]: These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-075343-fbf2cd7c-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/snapshot.js (hunk 13 above)
Comment (cockpituous) [inline src/libvirtApi/snapshot.js:66]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-075343-fbf2cd7c-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/snapshot.js (hunk 14 above)
Comment (cockpituous) [inline src/libvirtApi/snapshot.js:76]: These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-075343-fbf2cd7c-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/snapshot.js (hunk 15 above)
Comment (cockpituous) [inline src/libvirtApi/snapshot.js:82]: These 5 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-075343-fbf2cd7c-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/storagePool.js (hunk 16 above)
Comment (cockpituous) [inline src/libvirtApi/storagePool.js:56]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-075343-fbf2cd7c-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/storagePool.js (hunk 17 above)
Comment (cockpituous) [inline src/libvirtApi/storagePool.js:115]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-075343-fbf2cd7c-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/storagePool.js (hunk 18 above)
Comment (cockpituous) [inline src/libvirtApi/storagePool.js:117]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-075343-fbf2cd7c-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/storagePool.js (hunk 19 above)
Comment (cockpituous) [inline src/libvirtApi/storagePool.js:121]: These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-075343-fbf2cd7c-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/storagePool.js (hunk 20 above)
Comment (cockpituous) [inline src/libvirtApi/storagePool.js:136]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-075343-fbf2cd7c-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/storagePool.js (hunk 21 above)
Comment (cockpituous) [inline src/libvirtApi/storagePool.js:138]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-075343-fbf2cd7c-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/storageVolume.js (hunk 22 above)
Comment (cockpituous) [inline src/libvirtApi/storageVolume.js:85]: These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-075343-fbf2cd7c-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/common.js (hunk 1 above)
Comment (cockpituous) [inline src/libvirtApi/common.js:217]: These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-085010-77feb7f2-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/common.js (hunk 2 above)
Comment (cockpituous) [inline src/libvirtApi/common.js:228]: These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-085010-77feb7f2-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/common.js (hunk 3 above)
Comment (cockpituous) [inline src/libvirtApi/common.js:237]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-085010-77feb7f2-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/common.js (hunk 4 above)
Comment (cockpituous) [inline src/libvirtApi/common.js:241]: These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-085010-77feb7f2-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/common.js (hunk 5 above)
Comment (cockpituous) [inline src/libvirtApi/common.js:432]: These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-085010-77feb7f2-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/domain.js (hunk 23 above)
Comment (cockpituous) [inline src/libvirtApi/domain.js:698]: These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-085010-77feb7f2-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/interface.js (hunk 7 above)
Comment (cockpituous) [inline src/libvirtApi/interface.js:61]: These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-085010-77feb7f2-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/interface.js (hunk 8 above)
Comment (cockpituous) [inline src/libvirtApi/interface.js:73]: These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-085010-77feb7f2-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/network.js (hunk 9 above)
Comment (cockpituous) [inline src/libvirtApi/network.js:73]: These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-085010-77feb7f2-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/nodeDevice.js (hunk 11 above)
Comment (cockpituous) [inline src/libvirtApi/nodeDevice.js:60]: These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-085010-77feb7f2-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/nodeDevice.js (hunk 12 above)
Comment (cockpituous) [inline src/libvirtApi/nodeDevice.js:72]: These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-085010-77feb7f2-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/snapshot.js (hunk 13 above)
Comment (cockpituous) [inline src/libvirtApi/snapshot.js:66]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-085010-77feb7f2-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/snapshot.js (hunk 14 above)
Comment (cockpituous) [inline src/libvirtApi/snapshot.js:76]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-085010-77feb7f2-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/storagePool.js (hunk 16 above)
Comment (cockpituous) [inline src/libvirtApi/storagePool.js:56]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-085010-77feb7f2-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/storagePool.js (hunk 17 above)
Comment (cockpituous) [inline src/libvirtApi/storagePool.js:115]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-085010-77feb7f2-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/storagePool.js (hunk 18 above)
Comment (cockpituous) [inline src/libvirtApi/storagePool.js:117]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-085010-77feb7f2-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/storagePool.js (hunk 19 above)
Comment (cockpituous) [inline src/libvirtApi/storagePool.js:121]: These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-085010-77feb7f2-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/storagePool.js (hunk 20 above)
Comment (cockpituous) [inline src/libvirtApi/storagePool.js:136]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-085010-77feb7f2-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/storagePool.js (hunk 21 above)
Comment (cockpituous) [inline src/libvirtApi/storagePool.js:138]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-085010-77feb7f2-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/libvirtApi/storageVolume.js (hunk 22 above)
Comment (cockpituous) [inline src/libvirtApi/storageVolume.js:85]: These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1400-20240125-085010-77feb7f2-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

**Review by jelly — APPROVED** [review]
Thanks, that was some massive :factory_worker:

**Pull request merged**